lib/query_helper/sql_sort.rb

```diff
@@ -41,12 +42,12 @@ def attributes_sql_expression(sort_attribute)
           if direction == "desc"
             case ActiveRecord::Base.connection.adapter_name
             when "SQLite" # SQLite is used in the test suite
-              direction = "desc"
+              direction = @keep_nulls_last ? "desc nulls last" : "desc"
             else
               direction = "desc nulls last"
             end
           else
-            direction = "asc"
+            direction = @keep_nulls_last ? "asc nulls last" : "asc"
```

Comment:
@NileshN How do you enable this variable? It is by default false. 
If I want to support the asc nulls last then we need to send this parameter to the query_helper.update method from our application right?

Reply:
Good question @AjitKonde-Pattern. Here is the way to enable value to this attribute from the query-helper object:
`@query_helper.update(........., sql_sort: QueryHelper::SqlSort.new(
              sort_string: params[:sort],
              keep_nulls_last: true
            ))`

Reply:
@NileshN 
Solution to this. In query_helper(lib/query_helper.rb) update method, you need to add below code:
`@sql_sort.keep_nulls_last= keep_nulls_last if keep_nulls_last`

Reply:
In application, you just need pass the value:
@query_helper.update(query: query, keep_nulls_last: true)